### ISSUE TITLE
Crash after signing in (any website)

### ISSUE THREAD

**Issue by st3fan** [mozilla-mobile/android-components#5555]
```
01-13 13:27:51.781 30717 30717 E AndroidRuntime: Process: org.mozilla.fenix.nightly, PID: 30717
01-13 13:27:51.781 30717 30717 E AndroidRuntime: kotlin.NotImplementedError: Unable to validate saved logins. Are you using the correct class mozilla.components.concept.storage.NoopLoginValidationDelegate (Kotlin reflection is not available)?
01-13 13:27:51.781 30717 30717 E AndroidRuntime: 	at mozilla.components.feature.prompts.dialog.LoginDialogFragment$update$1.invokeSuspend(LoginDialogFragment.kt:29)
01-13 13:27:51.781 30717 30717 E AndroidRuntime: 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:2)
01-13 13:27:51.781 30717 30717 E AndroidRuntime: 	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:15)
01-13 13:27:51.781 30717 30717 E AndroidRuntime: 	at android.os.Handler.handleCallback(Handler.java:873)
01-13 13:27:51.781 30717 30717 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:99)
01-13 13:27:51.781 30717 30717 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:193)
01-13 13:27:51.781 30717 30717 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:6718)
01-13 13:27:51.781 30717 30717 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
01-13 13:27:51.781 30717 30717 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
01-13 13:27:51.781 30717 30717 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
```

**Comment by kbrosnan** [comment]
See also https://github.com/mozilla-mobile/fenix/issues/7625

**Comment by st3fan** [comment]
This was initially _after signing in with Mozilla SSO_ but I get the same crash after signing in to any website.

**Comment by Baron-Severin** [comment]
Closed by #5559
